recipes/zlib/1.2.11/conanfile.py

```diff
@@ -89,15 +89,6 @@ def _build_zlib(self):
             tools.replace_in_file('gzguts.h',
                                   '#if defined(_WIN32) || defined(__CYGWIN__)',
                                   '#if defined(_WIN32) || defined(__MINGW32__)')
```

Comment:
```suggestion
        if self.settings.compiler != "apple-clang" or tools.Version(self.settings.compiler.version) < "11":
```

Do not remove it, instead, add condition to skip apple-clang 12